core/server/data/meta/cover_image.js

```diff
@@ -1,11 +1,12 @@
 var config           = require('../../config'),
-    getContextObject = require('./context_object.js');
```

Comment:
some files have general changes for readability (core/server/data/meta/cover_image.js, core/server/data/meta/author_image.js, core/server/data/meta/author_image.js)
Could you please source them out as a separate PR to master?Thank you!